src/js/core/directives/ui-grid-header-cell.js

```diff
@@ -347,6 +347,24 @@
                 }).catch(angular.noop);
             }
 
+            function applyMoveColumn(moveRight) {
+              if (!$scope.col) { return; }
+              if (!uiGridCtrl.grid.api.colMovable) { return; }
```

Comment:
any reason not to combine these two early returns?

Reply:
It was just for readability.  I like this better than it anded.